### PR TITLE
added 2 enemies to assets and to prototype scene

### PR DESCRIPTION
**Pull request by enirfo**

### CODE REVIEW

File: scenes/PrototypeScene_01.tscn [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[gd_scene load_steps=15 format=3 uid="uid://bo1a11i38vuh7"]
+[gd_scene load_steps=17 format=3 uid="uid://bo1a11i38vuh7"]
```
Comment (enirfo) [inline scenes/PrototypeScene_01.tscn]: cose strane di godot. help

File: assets/3d/creatures/Blutschneke/Blutschneke_02.glb
Comment (dughiboogie) [inline assets/3d/creatures/Blutschneke/Blutschneke_02.glb:1]: Missing model scene. Create scene with imported model before committing asset.

File: assets/3d/creatures/Flugschneke/Flugschneke_01.glb
Comment (dughiboogie) [inline assets/3d/creatures/Flugschneke/Flugschneke_01.glb:1]: Missing model scene. Create scene with imported model before committing asset.

File: assets/3d/creatures/Flugschneke/Animations/fly.res
Comment (dughiboogie) [inline assets/3d/creatures/Flugschneke/Animations/fly.res:1]: Loop mode disabled. Interpolation should be set to `nearest`

File: assets/3d/creatures/Blutschneke/Animations/walk.res
Comment (dughiboogie) [inline assets/3d/creatures/Blutschneke/Animations/walk.res:1]: Loop mode disabled. Interpolation should be set to nearest

File: assets/3d/creatures/Flugschneke/Materials/Material.002.tres
Comment (dughiboogie) [inline assets/3d/creatures/Flugschneke/Materials/Material.002.tres:1]: Change material name.

File: assets/3d/creatures/Flugschneke/Materials/Material.003.tres
Comment (dughiboogie) [inline assets/3d/creatures/Flugschneke/Materials/Material.003.tres:1]: Change material name.

File: assets/3d/creatures/Flugschneke/Materials/purple worm side.tres
Comment (dughiboogie) [inline assets/3d/creatures/Flugschneke/Materials/purple worm side.tres:1]: Change material name to fit camelCase naming convention.

File: assets/3d/creatures/Flugschneke/Materials/purple worm.tres
Comment (dughiboogie) [inline assets/3d/creatures/Flugschneke/Materials/purple worm.tres:1]: Change material name to fit camelCase naming convention.

File: assets/3d/creatures/Blutschneke/Materials/purple worm.tres
Comment (dughiboogie) [inline assets/3d/creatures/Blutschneke/Materials/purple worm.tres:1]: Change material name to fit camelCase naming convention.

File: assets/3d/creatures/Blutschneke/Materials/purple worm side.tres
Comment (dughiboogie) [inline assets/3d/creatures/Blutschneke/Materials/purple worm side.tres:1]: Change material name to fit camelCase naming convention.

File: assets/3d/creatures/Ranaorso/240519_ranaorso_01.glb
Comment (dughiboogie) [inline assets/3d/creatures/Ranaorso/240519_ranaorso_01.glb:1]: Change name to fit camelCase naming convention.

File: scenes/PrototypeScene_01.tscn (hunk 1 above)
Comment (dughiboogie) [reply scenes/PrototypeScene_01.tscn]: help is on the way

File: assets/3d/creatures/Blutschneke/Blutschneke_02.glb
Comment (dughiboogie) [reply assets/3d/creatures/Blutschneke/Blutschneke_02.glb:1]: Change file version to fit semantic versioning convention (`Blutschneke_0.0.2`)

File: assets/3d/creatures/Flugschneke/Flugschneke_01.glb
Comment (dughiboogie) [reply assets/3d/creatures/Flugschneke/Flugschneke_01.glb:1]: Change file version to fit semantic versioning convention (Flugschneke_0.0.2)

File: scenes/PrototypeScene_01.tscn [hunk 2]
```diff
@@ -8,6 +8,8 @@
 [ext_resource type="Material" uid="uid://ddgo8827yv4h7" path="res://assets/3d/environment/rock/materials/rock.tres" id="4_l358p"]
 [ext_resource type="PackedScene" uid="uid://dwsdlvxqlwulo" path="res://assets/3d/environment/rock/240519_rock_medium_02.glb" id="6_a2tgg"]
 [ext_resource type="PackedScene" uid="uid://h4601tixrkix" path="res://assets/3d/environment/bush/240519_bush_bubbly_02.glb" id="8_rd8q8"]
+[ext_resource type="PackedScene" uid="uid://bdlecjgnvp85d" path="res://assets/3d/creatures/Blutschneke/Blutschneke_02.glb" id="9_gjq70"]
+[ext_resource type="PackedScene" uid="uid://c17ilaij47mhf" path="res://assets/3d/creatures/Flugschneke/Flugschneke_01.glb" id="10_jb8dr"]
```
Comment (dughiboogie) [inline scenes/PrototypeScene_01.tscn]: Update names

File: scenes/PrototypeScene_01.tscn
Comment (dughiboogie) [inline scenes/PrototypeScene_01.tscn:1]: Remove editable children from final scenes

**Review by dughiboogie — CHANGES_REQUESTED** [review]
A couple things to be updated following the latest conventions decisions.

Animations and creatures look dope af.

**Review by dughiboogie — APPROVED** [review]
Looks good!

**Pull request merged**